spec/api.yaml

```diff
@@ -112,25 +112,28 @@ paths:
                 v:
                   type: string
                   description: A base64URL-encoded bencoded DNS packet containing the DID Document.
-                retention_proof:
+                retention_solution:
                   type: string
-                  description: A retention proof calculated according to the spec-defined retention proof algorithm.
+                  description: A retention solution calculated according to the spec-defined retention solution algorithm.
```

Comment:
nice, I think it's good to avoid conflating the usage of `proof` across the two use cases:

1. cryptographic provenance using private keys
2. proof of expending energy (AKA PoW)

yeah I think we'd be wise to consolidate our usage of that word for (1) and not for (2)

also sets us up if we ever introduces a spam preventing mechanism other-than PoW